src/OpenTelemetry.Api/Context/Propagation/PropagationContext.cs

```diff
@@ -59,6 +67,29 @@ public PropagationContext(ActivityContext activityContext, Baggage baggage)
         /// <param name="right">Second Entry to compare.</param>
         public static bool operator !=(PropagationContext left, PropagationContext right) => !(left == right);
 
+        /// <summary>
+        /// Create a <see cref="PropagationContext"/> from a <see cref="Activity"/>.
+        /// </summary>
+        /// <param name="activity"><see cref="Activity"/>.</param>
+        /// <param name="baggage"><see cref="OpenTelemetry.Baggage"/>.</param>
+        /// <returns><see cref="PropagationContext"/>.</returns>
+        public static PropagationContext CreateFromActivity(Activity activity, Baggage baggage = default)
+        {
+            return new PropagationContext(activity, baggage);
+        }
```

Comment:
Why the static method vs. just making the new constructor public?

Reply:
Good question! Originally I had it like this:

```csharp
        public PropagationContext(ActivityContext activityContext, Baggage baggage)
        {
            this.activityContext = activityContext;
            this.Baggage = baggage;
        }

        public PropagationContext(Activity activity, Baggage baggage)
        {
            this.activityContext = activity?.Context ?? default;
            this.Baggage = baggage;
        }
```

Problem with that is anyone doing something like `var context = new PropagationContext(default, Baggage.Current)` suddenly gets an ambiguous method compiler error. Because it doesn't know which overload to pick for the `default` (could mean `default(ActivityContext)` or `default(Activity)`). I added the static to disambiguate basically.